packages/mdc-textfield/foundation.js

```diff
@@ -15,9 +15,11 @@
  * limitations under the License.
  */
 
+/* eslint-disable no-unused-vars */
```

Comment:
To ensure checking isn't disabled for the entire file, this line could be removed and a line-specific lint disable comment could be added above `MDCTextFieldHelperTextFoundation`

Reply:
Done! Here and in index.js